### ISSUE TITLE
Croptopia 1.17 on fabric 1.17 not working correctly?

### ISSUE THREAD

**Issue by ZappaNappa** [ExcessiveAmountsOfZombies/Croptopia#173]
**Version**
Which version of Croptopia are you using?
Croptopia-1.17-FABRIC-1.5.0

**Platform**
Which platform are you using Croptopia on?
(Fabric, Forge)
Fabric 1.17 (no snapshot or the new 1.17.1 the fabric launcher has)
Loader 0.11.6 (highest loader available on the fabric installer hmm.)

**Describe the issue**
A clear and concise description of what the issue is.
Unable to launch from vanilla minecraft launcher, not MultiMC or anything else.
Error code shown to me after launch latest.txt log
`[22:27:02] [main/INFO]: Loading for game Minecraft 1.17
[22:27:03] [main/FATAL]: A critical error occurred
net.fabricmc.loader.discovery.ModResolutionException: Could not find required mod: croptopia requires {fabric @ [>=0.36.0]}
	at net.fabricmc.loader.discovery.ModResolver.findCompatibleSet(ModResolver.java:198) ~[fabric-loader-0.11.6.jar:?]
	at net.fabricmc.loader.discovery.ModResolver.resolve(ModResolver.java:832) ~[fabric-loader-0.11.6.jar:?]
	at net.fabricmc.loader.FabricLoader.setup(FabricLoader.java:195) ~[fabric-loader-0.11.6.jar:?]
	at net.fabricmc.loader.FabricLoader.load(FabricLoader.java:185) [fabric-loader-0.11.6.jar:?]
	at net.fabricmc.loader.launch.knot.Knot.init(Knot.java:132) [fabric-loader-0.11.6.jar:?]
	at net.fabricmc.loader.launch.knot.KnotClient.main(KnotClient.java:28) [fabric-loader-0.11.6.jar:?]
Caused by: net.fabricmc.loader.util.sat4j.specs.ContradictionException: Creating Empty clause ?
	at net.fabricmc.loader.util.sat4j.minisat.constraints.cnf.Clauses.propagationCheck(Clauses.java:117) ~[fabric-loader-sat4j-2.3.5.4.jar:?]
	at net.fabricmc.loader.util.sat4j.minisat.constraints.cnf.Clauses.sanityCheck(Clauses.java:97) ~[fabric-loader-sat4j-2.3.5.4.jar:?]
	at net.fabricmc.loader.util.sat4j.minisat.constraints.MixedDataStructureDanielWL.createClause(MixedDataStructureDanielWL.java:81) ~[fabric-loader-sat4j-2.3.5.4.jar:?]
	at net.fabricmc.loader.util.sat4j.minisat.core.Solver.addClause(Solver.java:401) ~[fabric-loader-sat4j-2.3.5.4.jar:?]
	at net.fabricmc.loader.discovery.ModResolver.findCompatibleSet(ModResolver.java:195) ~[fabric-loader-0.11.6.jar:?]
	... 5 more`

**To Reproduce**
How can I reproduce the issue?
Run it? Sorry kinda new to this. 4 mods max

**Screenshots**
![Install setup on launcher](https://user-images.githubusercontent.com/41528369/124410133-cf7d1700-dd17-11eb-99e6-5230586f2e60.png)

**Other Mods**
What other mods are you using? Having a list might be helpful for me.
total 4
Croptopia-1.17-FABRIC-1.5.0
Fabric-API-1.17
fabric-language-kotlin-1.6.2+kotlin.1.5.20
Patchouli-1.17-54-FABRIC
![Mod List](https://user-images.githubusercontent.com/41528369/124410369-49150500-dd18-11eb-9e92-c35ba68d026e.png)

**Additional context**

![FabricLauncherError](https://user-images.githubusercontent.com/41528369/124410665-ef610a80-dd18-11eb-9e5d-71b19ac33eb4.png)
I even attempted to remove the fabric-language-kotlin jar I thought might have been needed by fabric itself but apparently not. 

I have no idea what the fabric version "fabric @ [>=0.36.0]" that the error is referring to.

**Comment by itschonky** [comment]
shouldnt you have "fabric-api-0.36.0+1.17.jar" (or something like that)? no clue what you got "Fabric-API-1.17" from
just replace it with the latest version from curseforge, heres a link to it: https://www.curseforge.com/minecraft/mc-mods/fabric-api/files/3358619

**Comment by ZappaNappa** [comment]
I couldn't remember where I found it from so I backtracked a few days and seems I got this file from 9minecraft source. Seems that's my biggest problem, sorry about that. The api was indeed the problem because it now runs with just croptopia, the new fabric api, and patchouli. Everything else was setup correctly except that. Now I gotta add in the other mods I have in slowly to see what's causing new conflicts. :) Have a good weekend.
